sdks/python/apache_beam/examples/inference/README.md

```diff
@@ -536,9 +536,9 @@ The pipeline reads rows of pixels corresponding to a digit, performs basic prepr
 The model is loaded from the saved model weights. This can be done by passing a function which creates the model and setting the model type as
 `ModelType.SAVED_WEIGHTS` to the `TFModelHandler`. The path to saved weights saved using `model.save_weights(path)` should be passed to the `model_path` argument.
 
-### Dataset and model for language modeling
+### Dataset and model for digit classification
 
-To use this transform, you need a dataset and model for language modeling.
+To use this transform, you need a dataset and model for digit classification.
```

Comment:
```suggestion
To use this transform, you need a dataset and model for MNIST digit classification.
```

Reply:
@AnandInguva 
I've made the requested changes and pushed the updated commit. Please take another look when you have a chance. Thank you for your feedback!